lib/SPIRV/SPIRVReader.cpp

```diff
@@ -739,6 +739,134 @@ SPIRVToLLVM::getMetadataFromNameAndParameter(std::string Name,
               ConstantInt::get(Type::getInt32Ty(*Context), Parameter))};
 }
 
+class IVDepMetadataEmitter {
+public:
+  using PointerSflnMapTy = std::map<Value *, unsigned>;
```

Comment:
This is a minor comment: IMHO, Sfln is confusing a little bit. Would it be more readable to use `PtrSafeLenMapTy` or `PointerSafeLenMapTy`?

Reply:
Fair point; besides, there have been inconsistencies between "safelen" and "sfln" in variable names. Fixed that in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/655f53345fa386251952842c11350e7e32c091e9.